src/amoc_networking.erl

```diff
@@ -1,37 +0,0 @@
--module(amoc_networking).
```

Comment:
I'm strongly against removing the module. It was (and may still be) used in case we have many network interfaces and want to use them all to open connections to the target node. I'd rather document this helper module with examples how to use it, instead of removing it.

Reply:
the idea was to move it to `amoc-arsenal-xmpp`